File: src/main/java/com/dh/ondot/member/domain/WithdrawalReason.java
package com.dh.ondot.member.domain;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Getter
@Builder
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor(access = AccessLevel.PRIVATE)
@Table(name = "withdrawal_reasons")
public class WithdrawalReason {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "withdrawal_reason_id")
    private Long id;

    @Column(name = "reason", nullable = false)
    private String reason;
}

File: src/main/kotlin/com/dh/ondot/member/domain/WithdrawalReason.kt
package com.dh.ondot.member.domain

import jakarta.persistence.*

@Entity
@Table(name = "withdrawal_reasons")
class WithdrawalReason(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "withdrawal_reason_id")
    val id: Long = 0L,

    @Column(name = "reason", nullable = false)
    val reason: String,
)
